### ISSUE TITLE
Tinkers Construct's EFLN won't fire break events 

### ISSUE THREAD

**Issue by Eufranio** [SpongePowered/SpongeForge#1209]
EFLN is a throwable item from Tinkers Construct mod (tconstruct:throwball) that explodes on impact. This explosion doesn't seem to be tracked, I use GriefPrevention (which cancells the explosion and break events) but using GPDebug shows that the blocks broken aren't firing the events (so GP cant catch them). This is a grief issue for servers running Tinkers.

**Comment by axle2005** [comment]
Duplicate of [1083](https://github.com/SpongePowered/SpongeForge/issues/1083)

**Comment by Eufranio** [comment]
Not sure if it's really duplicated, since my issue is because they BREAK but DONT FIRE the events. In this issue they don't even break...

**Comment by Aaron1011** [comment]
Thiis bug is due to the fact the Tinker's Construct performs custom explosions, which are spread out over multiple ticks. We're currently determining what the best way to address this issue is.

**Comment by phit** [comment]
The explosion event looks like a Sponge bug to me it fires `ExplosionEvent.Start` in Forge, but for some reason it never fires `ExplosionEvent.Pre` in Sponge.

Tested with:
Mantle-1.12-1.3.1.21.jar
spongeforge-1.12.2-2529-7.0.0-BETA-2741.jar
TConstruct-1.12-2.7.4.34.jar

Forge Test:
```java
    @SubscribeEvent
    public void onExplosionStart(ExplosionEvent.Start event) {
        System.out.println("ExplosionEvent.Start: " + event.toString());
    }
```
Sponge Test:
```java
    @Listener
    public void onExplosion(ExplosionEvent.Pre event) {
        System.out.println("ExplosionEvent.Pre: " + event.toString());
    }
```

![image](https://user-images.githubusercontent.com/2097483/33154936-5dcb3344-cfec-11e7-85ee-8dd9298c8d22.png)

only Forge posts the event


As for the break events
> Tinker's Construct performs custom explosions, which are spread out over multiple ticks

tinkers code https://github.com/SlimeKnights/TinkersConstruct/blob/7e6ea156bdc1d9af0b2e666b84c4f7a54d4ef6c9/src/main/java/slimeknights/tconstruct/gadgets/entity/EntityThrowball.java#L78

**Comment by StoneLegion** [comment]
Having this issue myself and googled it and looks like this is known so I'm happy about it. Not sure if it will get resolved in 1.10 in the near future but either way I'm one of those people :)

**Comment by phit** [comment]
this may have been fixed on tinkers side, can you retest @Eufranio
